terraform/deployments/apps/common.tf

```diff
@@ -1,11 +1,12 @@
 locals {
   asset_host                       = local.website_root
-  assets_url                       = "https://assets.${var.mesh_domain}"
-  content_store_url                = "https://content-store.${var.mesh_domain}"
+  assets_url                       = "https://static-ecs.test.govuk.digital"
+  frontend_assets_url              = "https://frontend.test.govuk.digital"
```

Comment:
If you intend to hardcode the test domain please add a TODO so that we can more easily find and fix it later. I think we already have a variable for `<env>.govuk.digital` somewhere (`public_lb_domain_name` I think - not the best name but it's there).

Reply:
No, I was spiking on assets and found it easier. I will put the correct format.